### ISSUE TITLE
feature: show error/warning when a value that does not exist(or not available) is used in template

### ISSUE THREAD

**Issue by jfboeve** [lightning-js/blits#242]
Hey, I ran into a snippet on Discord, where someone used a value in the template that was not declared. However there is no warning or error that shows it doesn't exist.

Snippet here: 

https://playground.lightningjs.io/#PgXV33lF9kin

**Comment by michielvandergeest** [comment]
@suresh-gangumalla this functionality has been added, right? Are we good to close this issue? Or are the next steps regarding this issue that it's better to leave it open?

**Comment by suresh-gangumalla** [comment]
Yes, @michielvandergeest  this functionality is fully added and now user can see warning message in console on Development mode.  I feel, we are good to close this.
